src/pages/[language]/blog/page/index.query.graphql

```diff
@@ -37,9 +37,24 @@ query Blog(
     slug
   }
   tag: tag(filter: { id: { eq: $tagId } }) {
+    i18nSlugs: _allSlugLocales {
+      locale
+      value
+    }
     id
     title
     slug
+    blogTopicSocial {
```

Comment:
Can this just be called 'social', or is it specifically for the blog section of the site?

Reply:
I dont like it either, but I wanted to be specific since it is not defined on a page model but on the tag model